cmd/yurt-tunnel-server/app/options/options.go

```diff
@@ -22,16 +22,17 @@ import (
 	"strings"
 	"time"
 
-	"github.com/spf13/pflag"
 	"k8s.io/client-go/informers"
 	"k8s.io/klog/v2"
 	"sigs.k8s.io/apiserver-network-proxy/pkg/server"
 
+	"github.com/spf13/pflag"
+
 	"github.com/openyurtio/openyurt/cmd/yurt-tunnel-server/app/config"
 	"github.com/openyurtio/openyurt/pkg/projectinfo"
+	"github.com/openyurtio/openyurt/pkg/util/certmanager"
 	"github.com/openyurtio/openyurt/pkg/yurttunnel/constants"
 	kubeutil "github.com/openyurtio/openyurt/pkg/yurttunnel/kubernetes"
-	"github.com/openyurtio/openyurt/pkg/yurttunnel/pki"
 )
 
```

Comment:
Sorry for my unclear expression, I meat that we should import package in such format
```go
	"fmt"
	"net"
	"strings"
	"time"

	"github.com/spf13/pflag"
	"k8s.io/client-go/informers"
	"k8s.io/klog/v2"
	"sigs.k8s.io/apiserver-network-proxy/pkg/server"

	"github.com/openyurtio/openyurt/cmd/yurt-tunnel-server/app/config"
	"github.com/openyurtio/openyurt/pkg/projectinfo"
	"github.com/openyurtio/openyurt/pkg/yurttunnel/constants"
	kubeutil "github.com/openyurtio/openyurt/pkg/yurttunnel/kubernetes"
	"github.com/openyurtio/openyurt/pkg/yurttunnel/pki"
```
First block should be golang official pkg
Second block should be the 3-rd part pkg
Third block should be the openyurt pkg

I just added a new CI lint check in #694. It will check the import package format. Maybe you can try `make lint` at yourt local host after it's merged.